src/pages/ReportSettingsPage.js

```diff
@@ -88,6 +88,7 @@ class ReportSettingsPage extends Component {
     }
 
     validateAndUpdatePolicyRoomName() {
+        Keyboard.dismiss();
```

Comment:
@Puneet-here Any reason for keeping it at the top of the function before validation. If not, let's keep it below the validation if statement and above `Report.updatePolicyRoomName`